src/styles/card-content.scss

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  margin-bottom: 12px;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  align-items: center;
+
+  @media screen and (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, $card-width);
+    }
+  }
+
+  @media screen and (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, $card-width);
+    }
+  }
+
+  @media screen and (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, $card-width);
+    }
+  }
+
+  &__card {
```

Comment:
Make the card a separate block (it's definitely a reusable part of CSS code)
if you need to set external styles for the card use BEM-mix (so you can still have `main__card` but with a small amount of props - only external styles like margin)